atc/api/jobs_test.go

```diff
@@ -1404,6 +1417,19 @@ var _ = Describe("Jobs API", func() {
 						"team_name": "some-team",
 						"start_time": 101,
 						"end_time": 200
+					},
+					{
+						"id": 3,
+						"name": "1.1",
```

Comment:
Might want to make this `1.2` for clarity.

Reply:
Wait why would that be 1.2?

Reply:
Because the rerun number is 2, not 1

Reply:
Oh the rerun number should be 1 then